okvis_ros2/src/RosbagReader.cpp

```diff
@@ -117,17 +120,19 @@ bool RosbagReader::startStreaming() {
   std::vector<int> numCamImages(numCameras_, 0);
   std::vector<int> numDepthImages(numCameras_, 0);
   for(const auto & info : metadata.topics_with_message_count) {
-    if(info.topic_metadata.name.compare("/okvis/imu0") == 0) {
+    if(info.topic_metadata.name.compare(parameters_.imu.name) == 0) {
       numImuMeasurements = info.message_count;
+      LOG (INFO) << "Reading imu topic: " << parameters_.imu.name;
```

Comment:
does this `LOG(x)` macro works with a space?


```suggestion
      LOG (INFO) << "Reading imu topic: " << parameters_.imu.name;
```
```suggestion
      LOG(INFO) << "Reading imu topic: " << parameters_.imu.name;
```